src/frontend/views/modules/user/users-autocomplete-input.pug

```diff
@@ -1,3 +1,3 @@
 .col-xs-12
-  tags-input.form-control(ng-model='mutableUsers', add-on-enter='{{!propagateEnterEvent}}', display-property='displayName', key-property='id', placeholder='{{ translatedPlaceholder }}', on-tag-adding='onAddingUser($tag)', on-tag-added='onUserAdded($tag)', on-tag-removed='onUserRemoved($tag)', add-from-autocomplete-only='{{::addFromAutocompleteOnly}}', replace-spaces-with-dashes='false')
+  tags-input.form-control(ng-model='mutableUsers', add-on-enter='{{!propagateEnterEvent}}', display-property='displayName', key-property='id', placeholder='{{ translatedPlaceholder || "Users" }}', on-tag-adding='onAddingUser($tag)', on-tag-added='onUserAdded($tag)', on-tag-removed='onUserRemoved($tag)', add-from-autocomplete-only='{{::addFromAutocompleteOnly}}', replace-spaces-with-dashes='false')
```

Comment:
By this, you do not have the translation for `Users` in other locales

Reply:
Fixed by you :trollface: Thanks